plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/StorageFacet.js

```diff
@@ -40,6 +40,7 @@ Ext.define('NX.coreui.view.repository.facet.StorageFacet', {
         xtype: 'fieldset',
         cls: 'nx-form-section',
         title: NX.I18n.get('ADMIN_REPOSITORIES_DETAILS_SETTINGS_STORAGE_FACET'),
+        hidden: true,
```

Comment:
Yes, at some point in the future we will want to be able to configure which backing BlobStore is used by a repository, but probably not for awhile.